src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(readCsvFile(fromFileName)));
+    }
 
+    private void writeToFile(String file, String data) {
+        File uptputFile = new File(file);
+        try {
+            Files.write(uptputFile.toPath(), data.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/5bae9e08-63c2-4211-9077-1996c56b8e39)

Reply:
Corrected.

Reply:
I can't see the correctness
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/fe7e602e-d4c3-44df-8bf1-37d143ba9226)